plugin.video.sarpur/sarpur/scraper.py

```diff
@@ -53,6 +69,20 @@ def get_media_url(page_url):
 
     return u"http://smooth.ruv.cache.is/{0}".format(sources[0][4:])
 
+def get_live_url(channel):
+    """
+    Finds a live stream url for channel
+    :param channel: Slug name of channel
+    :return An url
+    """
+    apiurl = "http://ruv.is/sites/all/themes/at_ruv/scripts/ruv-stream.php?format=json"    
+    try:
+        json = requests.get(apiurl + "&channel=" + channel)      
+    except:
+        return -1
+            
+    return json.json()['result'][1]
```

Comment:
What are you trying to achieve here? I don't see a json import... also FYI You should avoid using builtin names as identifiers. 

`json = requests.get(apiurl + "&channel=" + channel) ` 
could be 
`Jsonresults` = requests.get(apiurl + "&channel=" + channel)`

THX